polling_stations/apps/file_uploads/models.py

```diff
@@ -219,7 +218,6 @@ def send_confirmation_email(self):
         )
         email.send()
 
-    @transaction.atomic
     def send_error_email(self):
         subject = "File upload failed"
         message = f"File upload failure: {self}. Please investigate further."
```

Comment:
As far as I can tell from reading this function, this transaction wasn't really doing anything so I just removed it.